lib/models/blueprint.js

```diff
@@ -387,12 +388,15 @@ let Blueprint = CoreObject.extend({
     @method _checkInRepoAddonExists
     @param {String} inRepoAddon
   */
-  _checkInRepoAddonExists(inRepoAddon) {
-    if (inRepoAddon) {
-      let addon = findAddonByName(this.project, inRepoAddon);
+  _checkInRepoAddonExists(options) {
+    let addon;
+
+    if (options.inRepoAddon || options.in) {
+      addon = options.inRepoAddon ? findAddonByName(this.project, options.inRepoAddon) : findAddonByPath(this.project, options.in);
+
       if (!addon) {
```

Comment:
This error seems specific to the inRepoAdddon case. We should either change the message or guard against going down this path

Reply:
Already fixed that. Pushing now.

Reply:
Sorry, I thought you were referring to the error message. In terms of the error, we still want to check whether the in-repo addon exists. This is still a valid case regardless of the option used. Does that make sense?